charts/trino/templates/deployment-worker.yaml

```diff
@@ -144,7 +156,21 @@ spec:
             failureThreshold: {{ .Values.worker.readinessProbe.failureThreshold | default 6 }}
             successThreshold: {{ .Values.worker.readinessProbe.successThreshold | default 1 }}
           lifecycle:
+            {{- if .Values.worker.lifecycle }}
             {{- toYaml .Values.worker.lifecycle | nindent 12 }}
```

Comment:
I wonder if we should explicitly call `fail` here if `.Values.worker.gracefulShutdown.enabled` is true. If a custom lifecycle policy is used here, all the graceful shutdown configuration can also be applied manually.

Reply:
agreed